js/background.js

```diff
@@ -243,6 +245,10 @@ chrome.webRequest.onBeforeRequest.addListener(
 
                 updateBadge(e.tabId, tabs[e.tabId].dispTotal);
 
+                if(tabs[e.tabId].status === "complete"){
```

Comment:
This is the tab status that chrome sets. There are two statuses "loading" and "complete".